test_load fails on Windows with python=3.10

The *test_load* test is failing on Windows, python=3.10, but only when [run through GitHub Actions](https://github.com/csdms/bmi-topography/actions/runs/2820888645). It passes when run locally. It also passes on Windows, python<3.10, when run through Actions. 

This may be an upstream problem with *xarray* or *rioxarray*. Here's the [stack trace](https://github.com/csdms/bmi-topography/runs/7735044715?check_suite_focus=true):
```
================================== FAILURES ===================================
__________________________________ test_load __________________________________
Traceback (most recent call last):
  File "C:\Miniconda\envs\topography\lib\site-packages\xarray\core\common.py", line 284, in __setattr__
    object.__setattr__(self, name, value)
AttributeError: 'DataArray' object has no attribute '_file_obj'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\a\bmi-topography\bmi-topography\tests\test_topography.py", line 106, in test_load
    topo.load()
  File "D:\a\bmi-topography\bmi-topography\bmi_topography\topography.py", line 177, in load
    self._da = rioxarray.open_rasterio(_filename)
  File "C:\Miniconda\envs\topography\lib\site-packages\rioxarray\_io.py", line 775, in open_rasterio
    result._file_obj = manager
  File "C:\Miniconda\envs\topography\lib\site-packages\xarray\core\common.py", line 292, in __setattr__
    raise AttributeError(
AttributeError: cannot set attribute '_file_obj' on a 'DataArray' object. Use __setitem__ styleassignment (e.g., `ds['name'] = ...`) instead of assigning variables.
```

--- a/.github/workflows/build-test-ci.yml
+++ b/.github/workflows/build-test-ci.yml
@@ -40,7 +40,6 @@ jobs:
         run: pip install -e .
 
       - name: Test
-        if: ${{ !(matrix.os == 'windows-latest' && matrix.python-version == '3.10') }}
         env:
           OPENTOPOGRAPHY_API_KEY: ${{ secrets.OPENTOPOGRAPHY_API_KEY }}
         run: |

--- a/.github/workflows/build-test-ci.yml
+++ b/.github/workflows/build-test-ci.yml
@@ -51,5 +51,5 @@ jobs:
           bmi-test bmi_topography:BmiTopography --config-file=./examples/config.yaml --root-dir=./examples -vvv
 
       - name: Coveralls
-        if: matrix.os == 'ubuntu-latest' && matrix.python-version == "3.11"
+        if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.11'
         uses: AndreMiras/coveralls-python-action@v20201129

--- a/.github/workflows/build-test-ci.yml
+++ b/.github/workflows/build-test-ci.yml
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.9", "3.10", "3.11"]
 
     steps:
       - uses: actions/checkout@v2
@@ -46,10 +46,10 @@ jobs:
           pytest --cov=bmi_topography --cov-report=xml:./coverage.xml --cov-config=./setup.cfg -vvv
 
       - name: Test BMI
-        if: ${{ matrix.python-version == '3.9' }}
+        if: ${{ matrix.python-version == '3.9' }}  # stuck at py39 pending pymt update
         run: |
           bmi-test bmi_topography:BmiTopography --config-file=./examples/config.yaml --root-dir=./examples -vvv
 
       - name: Coveralls
-        if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.8'
+        if: matrix.os == 'ubuntu-latest' && matrix.python-version == "3.11"
         uses: AndreMiras/coveralls-python-action@v20201129

